README.md

```diff
@@ -9,3 +9,7 @@
 > scrndr = Screendoor(`API_KEY`)
 
 > responses = scrndr.get_project_responses(`PROJECT_ID`, {'per_page': 1, 'page' : 1}, 1)
```

Comment:
Would be nice if the documentation was a little more in depth.
Looking at this, I can figure out that PROJECT_ID is a screendoor id.  And that the second options parameter is a dictionary with a per_page and page property.  per_page is how many responses per page?  and page is the start page?  The third parameter of 1, I need to look into the source code to figure out.  From the function signature, it is a pages value - which means the number of pages to return?

Not sure if this sort of documentation goes here or maybe in a function header?

Reply:
agree that documentation could be improve. 

any preference on any of these? https://medium.com/@peterkong/comparison-of-python-documentation-generators-660203ca3804  also tagging @akegan for any suggestions. 

if no preference, might give pdoc a go.

Reply:
Gonna merge this PR in master.  Will create new PR for doc improvements.